crates/amaru/src/bin/amaru/main.rs

```diff
@@ -58,12 +58,19 @@ enum Command {
 struct Cli {
     #[command(subcommand)]
     command: Command,
+
     #[clap(long, action, env("AMARU_WITH_OPEN_TELEMETRY"))]
     with_open_telemetry: bool,
+
     #[clap(long, action, env("AMARU_WITH_JSON_TRACES"))]
     with_json_traces: bool,
+
+    #[arg(long, value_name = "STRING", default_value_t = DEFAULT_SERVICE_NAME.to_string())]
```

Comment:
Would it make sense to add `env` support, as e.g. `with_json_traces` and `AMARU_WITH_JSON_TRACES` ?